### ISSUE TITLE
User menu gone after upgrade to 2.6.1 from 2.5

### ISSUE THREAD

**Issue by gldrenthe89** [laravel/nova-issues#2094]
- Laravel Version: 6.4.1
- Nova Version: 2.6.1
- PHP Version: 7.3.8
- Database Driver and Version: MySQL 5.7.24
- Operating System and Version: Windows 10 (latest) and Ubuntu from Laravel Forge
- Browser type and version: Chrome latest

### Description

User menu gone after upgrading from 2.5 to 2.6.1

### Steps To Reproduce

composer update from 2.5 to 2.6.1

**Comment by jbrooksuk** [comment]
You need to re-publish the Nova assets and also run `php artisan vendor:publish` and select the Nova views.

**Comment by gldrenthe89** [comment]
> You need to re-publish the Nova assets and also run `php artisan vendor:publish` and select the Nova views.

Please reopen

**Comment by jbrooksuk** [comment]
You need to upgrade to v2.6.1.

**Comment by gldrenthe89** [comment]
> You need to upgrade to v2.6.1.

Didn't you reed the info above? i upgraded from 2.5 to 2.6.1 that's the whole point.

**Comment by jbrooksuk** [comment]
In which case, make sure that the published Nova views are correctly updated.

**Comment by gldrenthe89** [comment]
> In which case, make sure that the published Nova views are correctly updated.

Yes i have republished them and also did artisan view:clear even on a fresh deployement trough envoyer this problems exists

**Comment by jbrooksuk** [comment]
Specifically, can you check that [these changes](https://github.com/laravel/nova/commit/c2e0448986c23af336d8bf44cb02a74801f61c56) are in your published views?

**Comment by gldrenthe89** [comment]
i cannot see the link

> Specifically, can you check that [these changes](https://github.com/laravel/nova/commit/c2e0448986c23af336d8bf44cb02a74801f61c56) are in your published views?


I get a 404.

It looks like the $user variable hasn't been set

**Comment by gldrenthe89** [comment]
I did a php artisan nova:publish --force and it is working again but i lost al lot of changes.
How is it possible for me to see de breaking changes when a release is made. Shure i cannot --force everytime and show the diff then? isn't there a better way?

**Comment by jbrooksuk** [comment]
You need to publish the new layouts and do the diff.

**Comment by bolechen** [comment]
> > You need to publish the new layouts and do the diff.
> 
> Okay. thanks for the quick response

You need remove slot-scope and :handle-click in file `user.blade.php`

```
-<dropdown-trigger class="h-9 flex items-center" slot-scope="{toggle}" :handle-click="toggle">
 +<dropdown-trigger class="h-9 flex items-center">
```

**Comment by gldrenthe89** [comment]
> > > You need to publish the new layouts and do the diff.
> > 
> > 
> > Okay. thanks for the quick response
> 
> You need remove slot-scope and :handle-click in file `user.blade.php`
> 
> ```
> -<dropdown-trigger class="h-9 flex items-center" slot-scope="{toggle}" :handle-click="toggle">
>  +<dropdown-trigger class="h-9 flex items-center">
> ```

Spot on. Thanks. 

Would love a whay to see the diff before doing a upgrade. But guess that aint gonna come

**Comment by jbrooksuk** [comment]
> Would love a whay to see the diff before doing a upgrade. But guess that aint gonna come

This works like any other packages, you can always take a look in the diff on GitHub :)

**Comment by gldrenthe89** [comment]
> > Would love a whay to see the diff before doing a upgrade. But guess that aint gonna come
> 
> This works like any other packages, you can always take a look in the diff on GitHub :)

I know but where is the repo. i get a 404 on [https://github.com/laravel/nova/](https://github.com/laravel/nova/)

**Comment by davidhemphill** [comment]
Connect your Nova account here: https://nova.laravel.com/settings#github

**Comment by simonhamp** [comment]
`php artisan nova:publish` will not overwrite the views when upgrading. We've just bumped into this and it turns out that the only way to 'fix' this is to `php artisan nova:publish --force`... but this blows away any customisations.

For subtle changes where the fix is just editing/deleting a few lines of code, exploring a potentially large diff is a little wasteful.

It would be great if the Nova release notes (or somewhere else) could include a Laravel-docs-style Upgrade Guide when necessary to cover this as just saying "Don't forget to `php artisan nova:publish`" isn't enough.

I have a feeling this is what @gldrenthe89 was struggling to communicate.

It also shows that the 'fix' in v2.6.1 whatever that was didn't actually work.

**Comment by gldrenthe89** [comment]
> `php artisan nova:publish` will not overwrite the views when upgrading. We've just bumped into this and it turns out that the only way to 'fix' this is to `php artisan nova:publish --force`... but this blows away any customisations.
> 
> For subtle changes where the fix is just editing/deleting a few lines of code, exploring a potentially large diff is a little wasteful.
> 
> It would be great if the Nova release notes (or somewhere else) could include a Laravel-docs-style Upgrade Guide when necessary to cover this as just saying "Don't forget to `php artisan nova:publish`" isn't enough.
> 
> I have a feeling this is what @gldrenthe89 was struggling to communicate.
> 
> It also shows that the 'fix' in v2.6.1 whatever that was didn't actually work.

Exactly what i meant. For people like me who are developing apps with Nova, but do nog have access to the repo (the Nova login is from my boss) it would be great that when a --forced publish is needed this is told in the release. The i know i need to check my files.

**Comment by jbrooksuk** [comment]
To clarify it’s `php artisan nova:publish — force`

Replying from my phone but it should by dash dash.
